backend/libraries/ballerina-core/ballerina-core.fsproj

```diff
@@ -81,6 +82,11 @@
     <Compile Include="business rule engine/form engine/codegen/golang/language constructs/generatedTypes.fs" />
     <Compile Include="business rule engine/form engine/codegen/golang/language constructs/header.fs" />
     <Compile Include="business rule engine/form engine/codegen/golang/generator.fs" />
+    <Compile Include="codegen/python/model.fs" />

```

Comment:
are you sure you really need to list each and every file manually? not even a way to specify that an entire package (or similar) should be included?

Reply:
I don't know of any other way